connector-drawer/src/main/java/com/redhat/cloud/notifications/connector/drawer/DrawerPayloadBuilder.java

```diff
@@ -0,0 +1,82 @@
+package com.redhat.cloud.notifications.connector.drawer;
+
+import com.redhat.cloud.notifications.connector.drawer.constant.ExchangeProperty;
+import com.redhat.cloud.notifications.connector.drawer.model.DrawerEntry;
+import com.redhat.cloud.notifications.connector.drawer.model.DrawerEntryPayload;
+import com.redhat.cloud.notifications.connector.drawer.model.DrawerUser;
+import io.vertx.core.json.Json;
+import jakarta.enterprise.context.ApplicationScoped;
+import org.apache.camel.Exchange;
+import org.apache.camel.Message;
+import org.apache.camel.Processor;
+import org.apache.camel.util.json.JsonObject;
+import java.time.LocalDateTime;
+import java.util.List;
+import java.util.Map;
+
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.ORG_ID;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.OUTCOME;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.START_TIME;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.SUCCESSFUL;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.TARGET_URL;
+import static com.redhat.cloud.notifications.connector.ExchangeProperty.TYPE;
+import static java.time.ZoneOffset.UTC;
+
+@ApplicationScoped
+public class DrawerPayloadBuilder implements Processor {
+
+    public static final String CE_SPEC_VERSION = "1.0.2";
+    public static final String CE_TYPE = "com.redhat.console.notifications.drawer";
+
+    @Override
+    public void process(Exchange exchange) {
+        DrawerUser entry = exchange.getMessage().getBody(DrawerUser.class);
+        final DrawerEntryPayload entryPayloadModel = exchange.getProperty(ExchangeProperty.DRAWER_ENTRY_PAYLOAD, DrawerEntryPayload.class);
+        DrawerEntryPayload payloadToSend = new DrawerEntryPayload();
+        payloadToSend.setDescription(entryPayloadModel.getDescription());
+        payloadToSend.setCreated(entryPayloadModel.getCreated());
+        payloadToSend.setRead(entryPayloadModel.isRead());
+        payloadToSend.setSource(entryPayloadModel.getSource());
+        payloadToSend.setTitle(entryPayloadModel.getTitle());
+
+        payloadToSend.setId(entry.getDrawerNotificationUuid());
```

Comment:
Instead of storing the UUID in `DrawerUser`, could we do this and remove the UUID from `DrawerUser`?
```suggestion
        payloadToSend.setId(UUID.randomUUID());
```

Reply:
We need to store it because it will be sent back to engine to be persisted on database (this part is on the [next PR](https://github.com/RedHatInsights/notifications-backend/pull/2262/commits/80edd0682bff3c32a897841e82a937ffd65a4721))

Reply:
Okay, I don't see when that Id is used for now but I guess some code is missing.